docs/source/markdown/podman-generate-kube.1.md

```diff
@@ -19,6 +19,11 @@ Potential name conflicts between volumes are avoided by using a standard naming
 Note that if an init container is created with type `once` and the pod has been started, the init container will not show up in the generated kube YAML as `once` type init containers are deleted after they are run. If the pod has only been created and not started, it will be in the generated kube YAML.
 Init containers created with type `always` will always be generated in the kube YAML as they are never deleted, even after running to completion.
 
+*Note*: When using volumes and generating a Kubernetes YAML for an unprivileged and rootless podman container on an **SELinux enabled system**,  one of the following options must be completed:
+  * Add the "privileged: true" option to the pod spec
+  * Relabel the volume via the CLI command `chcon -t spc_t context -R <directory>`
```

Comment:
Nope the chcon should be contianer_file_t.

The spc_t would be to run with a securetyContext: seLinuxOptions of type:spc_t
```
        securityContext:
          seLinuxOptions:
            type: spc_t
```

Reply:
container_file_t is a file type; (Cat food, Dog food)
spc_t is a process type (domain) (Cat or Dog)

Reply:
oops, my bad. Fixed